css/style.css

```diff
@@ -0,0 +1,36 @@
+body{
```

Comment:
-pleas add this line of code for best CSS practice at the top of your code (Hint: to HTML):
box-sizing: border-box;
margin:0;
padding;